packages/embed/src/components/App.module.css

```diff
@@ -1,39 +1,11 @@
-@font-face {
-  font-family: "Avenir Next LT Pro";
-  font-style: normal;
-  font-weight: 500;
-  src: url("../assets/fonts/AvenirNextLTPro-Medium.ttf") format("truetype");
-}
-
-@font-face {
-  font-family: "Avenir Next LT Pro";
-  font-style: bold;
-  font-weight: 700;
-  src: url("../assets/fonts/AvenirNextLTPro-Demi-Bold.ttf") format("truetype");
-}
-
-@font-face {
-  font-family: "Avenir Next LT Pro";
-  font-style: bold;
-  font-weight: 900;
-  src: url("../assets/fonts/AvenirNextLTPro-Bold.ttf") format("truetype");
-}
-
-@font-face {
-  font-family: "Gilroy";
-  font-style: normal;
-  font-weight: 700;
-  src: url("../assets/fonts/Gilroy-Bold.otf") format("opentype");
-}
-
 body {
   margin: 0px;
   overflow: hidden;
 }
 
 .app {
   color: white;
-  font-family: "Avenir Next LT Pro";
+  font-family: "Avenir Next LT Pro" !important;
```

Comment:
does this get overridden somewhere? why `!important`?

Reply:
thanks this was debugging, will remove!